compiler/luci/pass/src/ResolveCustomOpAddPass.test.cpp

```diff
@@ -34,3 +170,61 @@ TEST_F(ResolveCustomOpAddPassTest, name)
   auto const name = _pass.name();
   ASSERT_NE(nullptr, name);
 }
+
+TEST_F(ResolveCustomOpAddPassTest, simple_test_CustomBroadcastTo)
+{
+  _g.init(true);
+
+  auto ret = _pass.run(_g.g());
+  EXPECT_EQ(true, ret);
+
+  auto add = dynamic_cast<luci::CircleAdd *>(_g.output(0)->from());
+  EXPECT_NE(nullptr, add);
+
+  // if this pass runs, x and y of Custom(AddV2) will be connected to x and y of Add.
+  // so, check if x and y of Custom(AddV2) Add are the equal to x and y of Add.
+  EXPECT_EQ(_g.input_x(), add->x());
+
+  // When adding two input values of differenct shapes, Add performs broadcasting according to NumPy
+  // rules. so, because BroadcastTo is skipped, BraodcastTo->input() will be connected to y of Add.
```

Comment:
This comment is something not what I expected.
Please fire a PR about this in another PR with a `NOTE` comment in Pass code.

Reply:
@seanshpark 
Okay. I will do that! :)

Reply:
I uploaded a new PR related to this.:) https://github.com/Samsung/ONE/pull/12615